### PR TITLE
Add Meson build definitions.

### PR DESCRIPTION
**Pull request by jpakkane**

### CODE REVIEW

File: upstream.wrap [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[wrap-file]
+subdirectory = imgui-sfml-2.1
```
Comment (legeana) [inline upstream.wrap]: upstream.wrap has directory: ☒

Should this be `directory` instead?

Comment (jpakkane) [reply upstream.wrap]: D'oh! :facepalm: Fixed.

**Comment by legeana** [comment]
Interesting. This wrap has a dependency. However I can't actually automatically test it as the dependency is not available in AUR and there is no `subproject/imgui.wrap`. This is another type of project and we will probably have more of these where it depends on other wraps but it doesn't sound like a good idea to actually ship another wrap here. What do you think?

**Comment by legeana** [comment]
https://github.com/mesonbuild/mesonwrap/issues/137

**Pull request merged**